pkg/manager/diff.go

```diff
@@ -414,16 +414,13 @@ func (kc *kernelContext) setupFuzzer(features flatrpc.Feature, syscalls map[*pro
 	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
 	corpusObj := corpus.NewFocusedCorpus(kc.ctx, nil, kc.coverFilters.Areas)
 	fuzzerObj := fuzzer.NewFuzzer(kc.ctx, &fuzzer.Config{
-		Corpus:   corpusObj,
-		Coverage: kc.cfg.Cover,
-		// TODO: it may be unstable between different revisions though.
-		// For now it's only kept true because it seems to increase repro chances in local runs (???).
-		FaultInjection: true,
```

Comment:
Please keep explicit `FaultInjection: false,` and add a comment as to why it's disabled.